src/web/utils/textHelpers.spec.ts

```diff
@@ -96,15 +99,39 @@ describe('Text helper tests', () => {
     });
   });
 
-  describe('sortstrings', () => {
+  describe('sort strings alphabetically', () => {
     const testStringArr = [
       ['atest', 'btest', 'ctest'],
       ['btest', 'atest', 'ctest'],
       ['ctest', 'btest', 'atest'],
     ];
     it.each(testStringArr)('should return array sorted alphabetically', () => {
       for (const t of testStringArr) {
-        expect(sortStrings(t)).toEqual(['atest', 'btest', 'ctest']);
+        expect(sortStringsAlphabetically(t)).toEqual(['atest', 'btest', 'ctest']);
+      }
+    });
+
+    const testStringArrNumbers = [['1test', '7test', '10test']];
```

Comment:
I think we can just simplify this to:
```
it('should return array sorted alphabetically', () => {
  const testStringArrNumbers = ['1test', '7test', '10test'];
    expect(sortStringsAlphabetically(testStringArrNumbers)).toEqual(['10test', '1test', '7test']);
  })
});
```

Reply:
And this probably applies to other methods below as well

Reply:
Done